charts/kargo/values.yaml

```
@@ -433,7 +433,10 @@ controller:
   ## @param controller.logLevel The log level for the controller.
   logLevel: INFO
 
-  ## @param controller.shardName [nullable] Set a shard name only if you are running multiple controllers backed by a single underlying control plane. Setting a shard name will cause this controller to operate **only** on resources with a matching shard name. Leaving the shard name undefined will designate this controller as the default controller that is responsible exclusively for resources that are **not** assigned to a specific shard. Leaving this undefined is the correct choice when you are not using sharding at all. It is also the correct setting if you are using sharding and want to designate a controller as the default for handling resources not assigned to a specific shard. In most cases, this setting should simply be left alone.
+  ## @param controller.isDefault When running multiple controllers backed by a single underlying control plane, designating this controller as the default will cause it to operate on resources not assigned to a specific shard. If controller.shardName is undefined, this controller will be considered the default **regardless** of the value of this field (as that was the behavior prior to the introduction of this field). If controller.shardName **is** defined, this controller will not be considered the default **unless, additionally** this field is true. i.e. A controller is effectively considered the default if !controller.shardName || controller.isDefault. If controller.shardName is defined **and** this field is true, this controller will operate **both** on resources explicitly assigned to it **as well as** those not assigned to a specific shard.
```

Does this support backtick blocks? If it does, I feel it might be better to do something like:

```suggestion
  ## @param controller.isDefault When running multiple controllers backed by a single underlying control plane, designating this controller as the default will cause it to operate on resources not assigned to a specific shard. If `controller.shardName` is undefined, this controller will be considered the default **regardless** of the value of this field (as that was the behavior prior to the introduction of this field). If `controller.shardName` **is** defined, this controller will not be considered the default **unless, additionally** this field is true. i.e. A controller is effectively considered the default if `!controller.shardName || controller.isDefault`. If controller.shardName is defined **and** this field is true, this controller will operate **both** on resources explicitly assigned to it **as well as** those not assigned to a specific shard.
```

to improve readability.

The same applies to the block below.